examples/divisions/division/country.yaml

```diff
@@ -0,0 +1,22 @@
+---
+id: example:division:country:us
+type: Feature
+geometry:
+  type: Point
+  coordinates: [-100.4458820, 39.7837304]
+properties:
+  theme: divisions
+  type: division
+  update_time: "2024-02-23T15:08:01Z"
+  version: 0
+  subtype: country
+  names:
+    primary: United States
+  country: US
+  hierarchies:
```

Comment:
Should countries have themselves in hierarchies or should they just have empty hierarchy?

Reply:
Good question. The [design document](https://wiki.overturemaps.org/x/nwEjAQ) suggested including the feature itself in the hierarchy, so this is being faithful to the design doc.

IMO the benefit of including it is its a nice consistent way to present the data. If someone wants to look at the full hierarchy for a feature, they don't have to construct it by inserting the feature into the hierarchy, they can just immediately see the whole thing at a glance.

Reply:
Ok, makes sense. 
I would be on the team "do not put feature itself into hierarchy", but I am ok either way.